presto-tests/src/main/java/com/facebook/presto/tests/datatype/CreateAndInsertDataSetup.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.datatype;
+
+import com.facebook.presto.tests.sql.SqlExecutor;
+import com.facebook.presto.tests.sql.TestTable;
+import com.google.common.base.Joiner;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.lang.String.format;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.IntStream.range;
+
+public class CreateAndInsertDataSetup
```

Comment:
maybe such `CreateAndInsertDataSetup` could be build from `CreateDataSetup` and `InsertDataSetup`